tests/codeceptjs/editors/array_test.js

```diff
@@ -21,6 +21,7 @@ Scenario('should trigger array (table) editing triggers', async (I) => {
 
   I.amAcceptingPopups();
   I.click('//button[contains(@class, "json-editor-btn-moveup") and @data-i="1"]');
+  I.wait(1)
```

Comment:
is there no `IwaitForPopup` or `...Element` or similar?

Reply:
there is no waitForPopUp yet unfortunately

Reply:
Would it be possible to write a custom webdriver helper? Similar to https://stackoverflow.com/a/31453019